src/desktop/apps/fair/routes.coffee

```diff
@@ -16,6 +16,8 @@ FilterArtworks = require '../../collections/filter_artworks'
 aggregationParams = require './components/browse/aggregations.coffee'
 InfoMenu = require '../../components/info_menu/index.coffee'
 
+DEFAULT_CACHE_TIME = 60
```

Comment:
It's hairy to override the `cacheTime` in each `fetch` call, as we make so many of them on this page and some of them even make nested calls. This uses a constant so at least the actual value is encapsulated in one place.